src/platform/haswell/platform.c

```diff
@@ -202,9 +202,9 @@ int platform_init(struct sof *sof)
 	trace_point(TRACE_BOOT_PLATFORM_CPU_FREQ);
```

Comment:
@xiulipan Patch looks fine, but I don't think it's correct to name this as a "typo". The same error is twice here, both in comment and in code, so does not seem like a typo, but simply a bug.

Reply:
OK, I will make it into a fix. Good point.